### ISSUE TITLE
Unable to subscript to sns topics. 

### ISSUE THREAD

**Issue by jab2805rakuten** [cloudposse/terraform-aws-cloudtrail#17]
Unable to subscript to sns topics.

like the aws cli example: 

```
aws cloudtrail create-subscription --name=awscloudtrail-example-jeff \
          --s3-new-bucket=awscloudtrail-new-bucket-example-jeff \
          --s3-prefix=prefix-example \
          --sns-new-topic=awscloudtrail-jeff
```

**Comment by osterman** [comment]
@jab2805rakuten this is a great use-case, however, currently the module doesn't implement anything for SNS. Pull Requests welcome and are reviewed promptly!

**Comment by Gowiem** [comment]
@jab2805rakuten this is now implemented as of version `0.20.0`. See #47 for more details
